docs/aca/05-aca-dapr-pubsubapi/index.md

```diff
@@ -184,8 +188,33 @@ Update below file in **TasksTracker.Processor.Backend.Svc** project.
 
 === "Program.cs"
 
-    ```csharp
-    --8<-- "docs/aca/05-aca-dapr-pubsubapi/Program.cs"
+    ```csharp hl_lines="9 13 15"
```

Comment:
I changed this from the import to inline, so that we can highlight the lines that need to be changed.